### DISCUSSION TITLE
找不到数据集duuie.zip请看这里

### DISCUSSION THREAD

**Discussion by Nathan-For** [PaddlePaddle/PaddleNLP#3307] [General]
如题，里面关于千言的数据集根本就不在项目里面，说是可以在比赛官网上找，但是根本就找不到，所以只是一个吹逼项目。

**Comment by linjieccc** [comment]
@Nathan-For 你好，数据集在[报名比赛](https://aistudio.baidu.com/aistudio/competition/detail/436/0/introduction)后即可下载。点击报名后，数据集介绍这里会提供下载链接
![image](https://user-images.githubusercontent.com/40840292/191025642-40e7f01d-c58c-4acb-892d-8a36253603a9.png)

**Reply by Nathan-For** [reply]
谢谢你的回复啊，我自己都没想到有人会回复的。 
![image](https://user-images.githubusercontent.com/55871906/191145900-1f241a67-afd9-4aeb-a38b-6698058f9604.png)
由于现在已经过了报名时间，所以无法下载数据集。
这真的是主办方非常严重的疏忽。

**Reply by Nathan-For** [reply]
大佬，你能不能送佛送到西，反馈一下这个问题，或者放个数据源出来？

**Reply by linjieccc** [reply]
可以点这里报名参加常规赛，常规赛链接：https://aistudio.baidu.com/aistudio/competition/detail/436/0/introduction

![image](https://user-images.githubusercontent.com/40840292/191155814-6e07628e-d2a3-44bc-911f-21bc9dd484d8.png)

**Reply by Nathan-For** [reply]
好的 终于找到了 谢谢你啊
